druntime/src/rt/sections.d

```diff
@@ -100,7 +102,6 @@ bool scanDataSegPrecisely() nothrow @nogc
         case "precise":
             return true;
         default:
-            __gshared err = new Error("DRT invalid scanDataSeg option, must be 'precise' or 'conservative'");
```

Comment:
> Move err to global scope in rt.sections.d to avoid the frontend lowering

Why is this needed?

Reply:
I was getting an error for calling the hook that calls `GC.malloc()` from this `@nogc` function. Now after rebasing it seems the error is fixed, so I rolled back this change.